pkg/fn/funcList.go

```diff
@@ -0,0 +1,27 @@
+package fn
+
+// Slice of functions to be executed after each other
+type FuncList struct {
```

Comment:
use `type FuncList []func()` instead of struct -> you can preinitialized size then

Reply:
Fixed